Project 1 - Creating GUI

```diff
@@ -0,0 +1,83 @@
+from tkinter import *
```

Comment:
You can either use line one that is = from tkinter import *
 that would import everything or only import whichever object you need as you did on line 2 and 3